resolver-dns/src/test/java/io/netty/resolver/dns/UnixResolverDnsServerAddressStreamProviderTest.java

```diff
@@ -139,6 +139,24 @@ public void searchDomainsWithMultipleSearch() throws IOException {
         assertEquals(Arrays.asList("linecorp.local", "squarecorp.local"), domains);
     }
 
+    @Test
+    public void searchDomainsWithMultipleSearchSeperatedByWhitespace() throws IOException {
+        File f = buildFile(
+                           "search linecorp.local squarecorp.local\n" +
```

Comment:
- Could merge the two lines or format like in `searchDomainsWithMultipleSearchSeparatedByTab`.
- How about merging the two tests into one by adding a `search` directive that contains both spaces and tabs (single space/tab, double space/tab, mixture of spaces and tabs, etc)?